appdata.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/linux/com.github.quaternion.appdata.xml b/linux/com.github.quaternion.appdata.xml
+index 84bca78..19d6446 100644
+--- a/linux/com.github.quaternion.appdata.xml
++++ b/linux/com.github.quaternion.appdata.xml
+@@ -19,7 +19,7 @@
+     <category>Network</category>
+   </categories>
+   
+-  <launchable type="desktop-id">quaternion.desktop</launchable>
++  <launchable type="desktop-id">com.github.quaternion.desktop</launchable>
```

Comment:
Why don't you just PR it to the [main Quaternion repo](https://github.com/quotient-im/Quaternion)?

Reply:
Because you already did it as I pointed out in the PR message, see your commit https://github.com/quotient-im/Quaternion/commit/1a53bf36e9a17c8aaead973b07ef6384dd98681e

Reply:
I can use that commit as patch if you want. I just made it before I found it was already done upstream. I'll do that tomorrow though because I was just about to sleep.

Reply:
I guess it will be fair to simply cherry-pick that commit into 0.0.9.4 and release 0.0.9.4g, otherwise releasing a Flatpak months after the official release would look dubious?

Reply:
Well, the update date shown on Flathub will change anyways because of the runtime bump so that's unavoidable really. I don't think people would care to see the discrepancy and if that were the case, they could just find the manifest and look for themselves that nothing is fishy. 

Feel free to make a release if you want and I'll use it though.

Reply:
Sorry, were you planning to make a release for this or can this be merged? It's the reason that the bump to 5.15 failed to build so it'd be good to fix that. :sweat_smile:

Reply:
Sorry, still planning to make a release %-) Should be there in an hour.

Reply:
Ok, turns out I'm no more in a good position to do 0.0.9.4 releases, since the CI tooling around 0.0.9.4 is no more in use, and a few other things around Quaternion have changed as well; so it seems prudent to just apply the patch and do a Flatpak for 0.0.9.4f, as you originally planned. Apologies for the confusion.

Meanwhile I'm finally unblocked on the 0.0.9.5 RC release path so hopefully I'll be able to ship it reasonably (...) soon and not migrate 0.0.9.4 to the new infra parts.